packages/typespec-azure-portal-core/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@azure-tools/typespec-azure-portal-core",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "tsc -p .",
+    "test": "mocha",
+    "test-official": "c8 mocha --forbid-only --reporter mocha-multi-reporters"
+  },
+  "keywords": [
+    "Azure",
+    "Portal",
+    "TypeSpec"
+  ],
+  "author": "",
+  "license": "MIT",
+  "type": "module",
+  "main": "dist/src/index.js",
+  "tspMain": "lib/main.tsp",
+  "peerDependencies": {
+    "@typespec/compiler": "~0.51.0"
+  },
+  "devDependencies": {
+    "typescript": "~5.3.3",
+    "@azure-tools/typespec-azure-core": "workspace:~0.37.1",
+    "@azure-tools/typespec-azure-resource-manager": "workspace:~0.37.0",
+    "@azure-tools/typespec-client-generator-core": "workspace:~0.37.0",
+    "@azure-tools/typespec-autorest": "workspace:~0.37.1",
+    "@types/mocha": "~10.0.6",
+    "@types/node": "~18.11.9",
+    "@typespec/compiler": "workspace:~0.51.0",
+    "@typespec/openapi": "workspace:~0.51.0",
+    "@typespec/http": "workspace:~0.51.0",
+    "@typespec/rest": "workspace:~0.51.0",
+    "@typespec/versioning": "workspace:~0.51.0",
```

Comment:
Do you need all those typespec dependencies?

Reply:
Yes I need all of them for my testing